### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lfundaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ more text
 
 
 hey jo !
-
+ff
```
Comment (lfundarotest) [inline README.md:8]: maybe change from ff to GG

**Review by lfundarotest — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (lfundaro) [reply README.md:8]: really ?

Comment (lfundaro) [reply README.md:8]: what about something else ?

Comment (lfundaro) [reply README.md:8]: what if we do ABC ?

Comment (lfundarotest) [reply README.md:8]: mmm not a good idea

Comment (lfundarotest) [reply README.md:8]: something else

Comment (lfundarotest) [reply README.md:8]: I have an idea

Comment (lfundarotest) [reply README.md:8]: another idea

Comment (lfundaro) [reply README.md:8]: i don't think so

**Pull request closed without merging**